client/src/components/related/Related.jsx

```diff
@@ -29,18 +29,22 @@ export default function Related({ feature }) {
         .then((results) => { setRelatedList(results.map((result) => result.data)); })
         .catch((err) => console.error(err));
     }
-  }, [feature]);
+  }, [feature.id]);
 
   return (
-    <div>
+    <div data-testid="related">
```

Comment:
```suggestion
    <div id="related-section">
```
Let's id our section containers. This will allow for us to more easily implement the Interactions API in the future.